cluster-autoscaler/simulator/drain.go

```diff
@@ -51,27 +50,26 @@ func FastGetPodsToMove(nodeInfo *schedulerframework.NodeInfo, skipNodesWithSyste
 		time.Now())
 
 	if err != nil {
-		return pods, blockingPod, err
+		return pods, daemonSetPods, blockingPod, err
 	}
 	if pdbBlockingPod, err := checkPdbs(pods, pdbs); err != nil {
-		return []*apiv1.Pod{}, pdbBlockingPod, err
+		return []*apiv1.Pod{}, []*apiv1.Pod{}, pdbBlockingPod, err
 	}
 
-	return pods, nil, nil
+	return pods, daemonSetPods, nil, nil
 }
 
-// DetailedGetPodsForMove returns a list of pods that should be moved elsewhere if the node
+// DetailedGetPodsForMove returns a list of pods that should be moved elsewhere and a list of DaemonSet pods that should be evicted if the node
```

Comment:
Nit: the line length in the func comments seems to extend a bit long. See if you can wrap it.